src/components/ContentHighlights/ContentHighlightArchivedCoursesAlert.jsx

```diff
@@ -0,0 +1,53 @@
+import { Alert } from '@edx/paragon';
+import { useContext, useState } from 'react';
+import dayjs from 'dayjs';
+import { NEW_ARCHIVED_COURSE_ALERT_DISMISSED_COOKIE_NAME, archivedHighlightsCoursesCookies } from './data/constants';
+import { EnterpriseAppContext } from '../EnterpriseApp/EnterpriseAppContextProvider';
+import { enterpriseCurationActions } from '../EnterpriseApp/data/enterpriseCurationReducer';
+
+const ContentHighlightArchivedCoursesAlert = () => {
+  const archivedCoursesCookies = {};
+  const [isOpen, setIsOpen] = useState(true);
+
+  const { enterpriseCuration: { enterpriseHighlightedContents, dispatch } } = useContext(EnterpriseAppContext);
+  enterpriseHighlightedContents.forEach(content => {
+    content.highlightedContent.forEach(course => {
+      if (course.courseRunStatuses?.length === 1 && course.courseRunStatuses.includes('archived')) {
```

Comment:
Should we be checking if all the courseRunStatuses are archived here?

nit: `'archived'` should live in a constant somewhere